teste/SME.SGP.TesteIntegracao/AuditoriaAdministrador/Ao_gerar_aula_com_auditoria_administrador.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Shouldly;
+using SME.SGP.Aplicacao.Interfaces;
+using SME.SGP.Dominio;
+using SME.SGP.Dominio.Interfaces;
+using SME.SGP.Infra;
+using SME.SGP.TesteIntegracao.Setup;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace SME.SGP.TesteIntegracao
+{
+    public class Ao_gerar_aula_com_auditoria_administrador : TesteBase
```

Comment:
Tem que criar um teste de inserção simulando a chamada do endpoint e outro uma chamada mensagem processada pelo Worker para garantir que a implementação esta funcionando.

Forçar a inclusão de um Administrador no Claim não garante nada.